lib/public/views/Logs/Logs.js

```diff
@@ -50,6 +53,70 @@ export default class Overview extends Observable {
         this.notify();
     }
 
+    /**
+     * Create the log with the variables set in the model, handle errors appriopately
+     * @returns {undefined}
+     */
+    async createLog() {
+        const response = await fetchClient('api/logs', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                title: this.getTitle(),
+                text: this.getText(),
+            }),
+        });
+
+        const result = await response.json();
```

Comment:
The response is valid when it has the data property containing the created log (useful for a redirect). The response (or request) is invalid if it has the error property, this should allow for better error reporting.

Reply:
The error handling has been improved as per this comment.